cEP-0021.md

```diff
@@ -0,0 +1,265 @@
+# Debug and Profile Bears
+
+| Metadata |                                                   |
+| -------- | ------------------------------------------------- |
+| cEP      | 0019                                              |
+| Version  | 1.0                                               |
+| Title    | Debug and Profile Bears                           |
+| Authors  | Vaibhav Kumar Rai <mailto:raivaibhav08@gmail.com> |
+| Status   | Proposed                                          |
+| Type     | Process                                           |
+
+## Abstract
+
+This cEP describes the detailed implementation of profiling interface as a part
+of the
+[GSoC'18 project](https://summerofcode.withgoogle.com/projects/#6109762077327360).
+
+## Introduction
+
+Profiling a Python program is doing a dynamic analysis that measures the
+execution time of the program and everything that compose it. That means
+measuring the time spent in each of its functions. This will give the data about
+where our program is spending time, and what area might be worth optimizing,
+thus we can provide an *Profiling interface* from within coala and make it
+available to bear writers so that they can create Better Performant bears.
+
+This proposal is to introduce the Profiling interface so that bear writer will
+have the ability to jump directly into Bear code and move through it using a
+debugger interface and Profile the Bear code to create Better Performant
+bears.
+
+## Proposed Change
+
+Bear writer can invoke Profiler through *CLI*:
+
+- By passing custom settings through *CLI*
+  `coala -b Bear Mybear --files file --profile-bears -"True,...."`,
```

Comment:
`-"True,..."` . Is the `-` intended there?

Reply:
No it's my mistake I am correcting that :+1: